Backend/Middleware-APD/src/main/java/middlewareapd/util/JwtUtil.java

```diff
@@ -0,0 +1,66 @@
+package middlewareapd.util;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.auth0.jwt.exceptions.JWTVerificationException;
+import com.auth0.jwt.interfaces.DecodedJWT;
+
+import middlewareapd.exception.UnauthorizedException;
+
+import java.util.Date;
+
+/**
+ * JwtUtil is a utility class that provides methods for creating and verifying JSON Web Tokens (JWTs).
+ * It handles the generation of tokens and their validation using a secret key.
+ */
+public class JwtUtil {
+
+    // Secret key for signing the JWT
+    private static final String SECRET_KEY = "your_secret_key_here";
+
+    // Expiration time for the token (e.g., 10 minutes in milliseconds)
+    private static final long EXPIRATION_TIME = 600_000;
+
+    /**
+     * Generates a JWT token for the given email, UUID, and admin status.
+     *
+     * @param email   the email address to be used as the subject of the token
+     * @param uuid    a unique identifier for the user
+     * @param isAdmin an integer indicating whether the user has admin privileges (0 or 1)
+     * @return the generated JWT token as a String
+     */
+    public static String generateToken(String email, String uuid, int isAdmin) {
+        // Define the signing algorithm using the secret key
+        Algorithm algorithm = Algorithm.HMAC256(SECRET_KEY);
+
+        // Create the token
+        String token = JWT.create()
+                .withSubject(email) // Set the email as the subject (username)
+                .withClaim("uuid", uuid) // Add UUID to the token
+                .withClaim("isAdmin", isAdmin) // Add isAdmin to the token
+                .withIssuedAt(new Date()) // Set issued time (now)
+                .withExpiresAt(new Date(System.currentTimeMillis() + EXPIRATION_TIME)) // Set expiration time
```

Comment:
can add the expiration time in like hours

Reply:
I currently set the time for 10min (since we kinda not running this for very long) so I changed how EXPIRATION_TIME was declared (min * num of sec in min * num of ms in a sec) then added a var to declare num of minutes